docs/en/basic/Web-Interface.md

```diff
@@ -118,7 +118,7 @@ but shows information specific to the particular worker.
 ### BlockInfo Page
 
 In the "BlockInfo" page, information on the files stored by the worker is shown,
-such as the file size and which tiers the file is stored on.
+such as the file size and which tier the file is stored on.
```

Comment:
It's technically possible for blocks to be stored in multiple tiers across different workers. Will the UI show both?

Reply:
I'm not sure, is there a way I can manually store blocks on multiple tiers to test?

The UI code only shows 'Tier', so I assumed it was single.

Reply:
If the UI only specifies "Tier" for now, then I think we can leave it at that.